core/util.py

```diff
@@ -65,3 +65,20 @@ def get_project_number(project_id):
     :return: project number
     """
     return int(project_id.split("-")[2])
+
+
+def construct_index(section: str, column: str, rows: tuple[int] | int) -> str:
+    """Constructs the index of an error from the column and rows it occurred in.
+
+    :param section: the internal table name where the error occurred
+    :param column: the internal table name where the error occurred
+    :param rows: tuple or int of row indexes where the error occurred
+    :return: indexes tuple of constructed letter and number indexes
+    """
+
+    column_letter = TABLE_AND_COLUMN_TO_ORIGINAL_COLUMN_LETTER[section][column]
+    if type(rows) is int:
+        indexes = column_letter + str(rows)
+    else:
+        indexes = ", ".join([column_letter + str(row) for row in rows])
+    return indexes
```

Comment:
Can we have some unit tests for this function

Reply:
added to test_failures for want of a better place to put them. As that seems like this function will be used the most